Source/FieldSolver/ImplicitSolvers/ImplicitSolver.cpp

```diff
@@ -121,3 +121,83 @@ void ImplicitSolver::InitializeMassMatrices ()
     }
 
 }
+
+void ImplicitSolver::PreRHSOp ( amrex::Real  a_cur_time,
+                                int          a_nl_iter,
+                                bool         a_from_jacobian )
```

Comment:
```suggestion
void ImplicitSolver::PreRHSOp ( const amrex::Real  a_cur_time,
                                const int          a_nl_iter,
                                const bool         a_from_jacobian )
```

These can be `const`